[CC 1.0.0] Refactored changes in CC docs (#5119)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/choreo-connect-analytics.md

[![Choreo Connect Analytics Architecture]({{base_path}}/assets/img/deploy/mgw/choreo-connect-analytics-architecture.png)]({{base_path}}/assets/img/deploy/mgw/choreo-connect-analytics-architecture)

> There are two main components related to internal gRPC request for sending `StreamAccessLogsMessage` from `router` to `enforcer`. Below section explains those two components which are used to collect analytics data.

Reviewer: ```suggestion
There are two main components related to internal gRPC request for sending `StreamAccessLogsMessage` from `router` to `enforcer`. The latter mentioned two components, which are used to collect analytics data, are explained in the following sections.
```
